jupyter_console/ptshell.py

```diff
@@ -135,6 +135,10 @@ class ZMQTerminalInteractiveShell(SingletonConfigurable):
         help="Override highlighting format for specific tokens"
     )
 
+    true_color = Bool(False, config=True,
+        help="Use 24bit colors instead of 256 colors in prompt highlighting"
```

Comment:
What's the status of terminal support? Can we give users looking at the help some indication of whether their terminal is likely to support it?